### PR TITLE
add progress bar to `alchemy` docking (`prep run`)

### PR DESCRIPTION
**Pull request by JenkeScheen**
fixes https://github.com/asapdiscovery/asapdiscovery/issues/1202

## Developers certificate of origin
- [x] I certify that this contribution is covered by the MIT license as defined in our [LICENSE](https://github.com/choderalab/asapdiscovery/blob/main/LICENSE) and adheres to the [**Developer Certificate of Origin**](https://developercertificate.org/).

### CODE REVIEW

**Comment by JenkeScheen** [comment]
I'm going to also add a progress bar to the charging stage, and work in some exception catches that I run into for some large sets that we're running

File: asapdiscovery-docking/asapdiscovery/docking/schema/pose_generation.py [hunk 1]
```diff
@@ -742,31 +782,13 @@ def _generate_poses(
...
-                # make sure properties at the top level get added to the conformers
-                sd_tags = get_SD_data(openeye_mol)
-                set_SD_data(openeye_mol, sd_tags)
-
-                if posed_ligand.GetNumConformers() > 0:
-                    # save the mol with all conformers
-                    result_ligands.append(openeye_mol)
-                else:
-                    failed_ligands.append(openeye_mol)
+                except RuntimeError:
```
Comment (hmacdope) [inline asapdiscovery-docking/asapdiscovery/docking/schema/pose_generation.py:790]: Add progress bar in same way to non-parallel calculation.

**Review by hmacdope — CHANGES_REQUESTED** [review]
Just a few drive by comments, looking good otherwise.

**Comment by JenkeScheen** [comment]
@hmacdope I'm going to tackle ligand outlier catching in another PR, let's keep this PR simple by only adding progress bars.

File: asapdiscovery-docking/asapdiscovery/docking/schema/pose_generation.py [hunk 2]
```diff
@@ -464,6 +471,7 @@ def _generate_poses(
                     failed_ligands.append(posed_ligand)
                 else:
                     result_ligands.append(posed_ligand)
+                progressbar.update(1)
```
Comment (hmacdope) [inline asapdiscovery-docking/asapdiscovery/docking/schema/pose_generation.py:474]: Not defined on this branch.

**Review by hmacdope — APPROVED** [review]
LGTM

**Pull request merged**